gateway/ln-gateway/src/envs.rs

```diff
@@ -1,50 +1,75 @@
-// Env variable to TODO
-pub const FM_CLN_EXTENSION_LISTEN_ADDRESS_ENV: &str = "FM_CLN_EXTENSION_LISTEN_ADDRESS";
-
-// Env variable to TODO
+// Environment variable that specifies the directory of the gateway's database.
 pub const FM_GATEWAY_DATA_DIR_ENV: &str = "FM_GATEWAY_DATA_DIR";
 
-// Env variable to TODO
+// Environment variable that specifies the address the gateway's HTTP server
+// should listen on.
 pub const FM_GATEWAY_LISTEN_ADDR_ENV: &str = "FM_GATEWAY_LISTEN_ADDR";
```

Comment:
Since you are already touching the comments, can you make them doc comments please (three slashes)?

Reply:
Done